app/helpers/constants.js

```diff
@@ -0,0 +1,8 @@
+const CONSTANTS = {
+    DBVKT_DIRECTORY: '.dbvtk',
```

Comment:
If you want to use the dbptk home folder in this application, you should pass this directory as an argument to the process that starts dbptk.

Reply:
Done.